Makefile

```diff
@@ -18,6 +18,9 @@ run-django:
 manage:
 	SECRET_KEY=pah-fm DJANGO_SETTINGS_MODULE=pah_fm.settings python3 backend/manage.py ${CMD}
 
+populate-database:
+	docker-compose exec backend python3 manage.py populate_database
```

Comment:
`exec` will fail if container is not running
I'd prefer `docker-compose rm --rm`